packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -215,14 +215,22 @@ class _BottomSheetState extends State<BottomSheet> {
   }
 
   void _handleDragUpdate(DragUpdateDetails details) {
-    assert(widget.enableDrag);
+    assert(
```

Comment:
If I move this assertion to the 'BottomSheet''s constructor, it will break some tests.